policy/cnspec_policy.proto

```diff
@@ -645,7 +653,7 @@ message ScoreDelta {
 message ScoredRiskFactor {
   string mrn = 1;
   float risk = 2;
-  bool is_absolute = 3;
```

Comment:
I know, we save a risk factor struct in the backend. Do you know whether it is this one? Would this break the unmarshal of already saved risk factors?